torch/_inductor/ir.py

```diff
@@ -220,6 +221,11 @@ class IndexingDiv(sympy.Function):
 
     nargs = (2,)
 
+    def _sympystr(self, printer):
+        base = printer.parenthesize(self.args[0], 50)
+        divisor = printer.parenthesize(self.args[1], 50)
+        return f"{base}//{divisor}"
```

Comment:
@nunoplopes fixed this properly in shape envs copy, should follow how his fix goes

Reply:
Umm... I'm not totally sure what you're referring to here.

Oh, are you referring to the precedence stuff?

Reply:
Yep, You have to declare a precedence field so the other operations that use this one as argument can parameterize it appropriately.

Reply:
see here: https://github.com/pytorch/pytorch/commit/2b0abd4ce38d049295dba2ba84ee2280aff7fa0d#diff-03557db7303b8540f095b4f0d9cd2280e1f42f534f67d8695f756ec6c02d3ec7